### PR TITLE
responsiveness done

### PR DESCRIPTION
**Pull request by mahenoorsalat**
**Pull request : Make the design compatible with different devices**

- Fixes issue #2 

**Changes made :**

- [X] Updated the navbar to include a hamburger menu for mobile screens, allowing the menu items to collapse into a dropdown, enhancing usability on smaller devices.

- [X] Modified the input field to use w-full and adjusted padding for better responsiveness across various screen sizes, ensuring it fits the container appropriately.

- [X] Enhanced the overall user interface by streamlining padding values for both the navbar and input field, providing a more consistent and user-friendly experience.

**Let me know if any other touch are needed! Thank you 👍**

Ping @mohitahlawat2001 

responsive1 

 After clicking it

### CODE REVIEW

**Comment by mahenoorsalat** [comment]
The issue with running the web application was caused by the presence of a .babelrc file in the project directory. I temporarily moved this file outside of the repository, and the application is now functioning smoothly.

Should I proceed to remove the .babelrc file entirely and push my changes to the repository? @mohitahlawat2001

**Comment by mohitahlawat2001** [comment]
@mahenoorsalat  did you try run "npm run  build" as it is still failing to deploy , 

try to run 
`npm run build`
and see what error you getting

**Comment by mahenoorsalat** [comment]
```yaml
@parcel/transformer-babel: Parcel includes transpilation by default. Babel config .babelrc contains only redundant presets. Deleting it may significantly      
improve build performance.

  C:\HactoberFest Loco-Musica\Loco-Musica\.babelrc:2:18
    1 | {
  > 2 |     "presets": [["@babel/preset-env", {"targets": {"node": "current"}}],
  >   |                  ^^^^^^^^^^^^^^^^^^^
  > 3 |                 ["@babel/preset-react", {"runtime": "automatic"}]
  >   |                  ^^^^^^^^^^^^^^^^^^^^^
    4 |     ]
    5 | }

  💡 Delete .babelrc
  📝 Learn more: https://parceljs.org/languages/javascript/#default-presets

@parcel/transformer-babel: @babel/preset-env does not support Parcel's targets, which will likely result in unnecessary transpilation and larger bundle sizes. 

  C:\HactoberFest Loco-Musica\Loco-Musica\.babelrc:2:18
    1 | {
  > 2 |     "presets": [["@babel/preset-env", {"targets": {"node": "current"}}],
  >   |                  ^^^^^^^^^^^^^^^^^^^
    3 |                 ["@babel/preset-react", {"runtime": "automatic"}]
    4 |     ]

  💡 Either remove @babel/preset-env to use Parcel's builtin transpilation, or replace with @parcel/babel-preset-env
  📝 Learn more: https://parceljs.org/languages/javascript/#custom-plugins
 ```

File: src/components/Header.js [hunk 1]
```diff
@@ -1,95 +1,100 @@
...
+              <Link className="hover:text-pink-300 transform transition-transform duration-300 scale-100 hover:text-xl" to="/">Home</Link>
+            </li>
+            <li>
+              <Link className="hover:text-pink-300 transform transition-transform duration-300 scale-100 hover:text-xl" to="/about">About</Link>
             </li>
             <li>
-              <Link className="flex" to="/recipeStore">
-              <img src={RecipeStore} alt="recipeStore" className="h-5 w-5 mx-2" /> -
-               <span className="mx-2">{recipes.length}</span>
+              <Link className="hover:text-pink-300 transform transition-transform duration-300 scale-100 hover:text-xl" to="/contact">Contact</Link>
```
Comment (mohitahlawat2001) [inline src/components/Header.js:63]: on hover not need to change test size , only make highlight

File: src/components/Cart.js [hunk 2]
```diff
@@ -1,108 +1,114 @@
...
-          </div>
-        ))}
-      </div>
 
-      <div className="mt-4">
-        <h2 className="text-2xl font-bold">Total: ₹{totalPrice() / 100}</h2>
-      </div>
-    </div>
-  );
+            <div className="grid grid-cols-3 gap-4">
```
Comment (mohitahlawat2001) [inline src/components/Cart.js:68]: it does not reflecting when i increase item amount from menu itself , it working individually which is wrong

[live site](https://loco-musica.vercel.app/cart)

go to site try add item to chart then click to + icon to increase
it is reflecting in cart ,

but with your it is not reflecting in card

**Review by mohitahlawat2001 — CHANGES_REQUESTED** [review]

File: src/components/Header.js (hunk 1 above)
Comment (mahenoorsalat) [reply src/components/Header.js:63]: okay

File: src/components/Body.js [hunk 3]
```diff
@@ -84,18 +84,28 @@ const Body = () => {
 
   return (
     <>
-      <div className="p-4 text-center rounded-xl mx-40 ">
+
```
[the remark this answers is not in the document]
Comment (mahenoorsalat) [reply src/components/Body.js:87]: that thing comes when I did pull

File: src/components/Cart.js (hunk 2 above)
Comment (mahenoorsalat) [reply src/components/Cart.js:68]: okay I will reupdate it with your code only

**Comment by mahenoorsalat** [comment]
```yaml
import GoogleLogin from "./GoogleLogin";
```

I just commented this line in login.js as it create an error with live website
I didn't why this create an issue as it related to backend work ..


please look at this issue 
Thank you :)

**Comment by mohitahlawat2001** [comment]
thank you for pr 


> make sure you follow and star ⭐ repo

**Pull request merged**